ext/B/t/optree_for.t

```diff
@@ -0,0 +1,191 @@
+#!perl
+
+BEGIN {
+    unshift @INC, 't';
+    require Config;
+    if (($Config::Config{'extensions'} !~ /\bB\b/) ){
+        print "1..0 # Skip -- Perl configured without B module\n";
+        exit 0;
+    }
+    if (!$Config::Config{useperlio}) {
+        print "1..0 # Skip -- need perlio to walk the optree\n";
+        exit 0;
+    }
```

Comment:
useperlio cannot be disabled, I suspect this was copied from an existing ext/B test.

Reply:
Yes. See #19126

Reply:
Gone in the rebase